trio_run_in_process/exceptions.py

```diff
@@ -8,3 +8,16 @@ class ProcessKilled(BaseRunInProcessException):
 
 class InvalidState(BaseRunInProcessException):
     pass
+
+
+class UnpickleableValue(BaseRunInProcessException):
```

Comment:
Looking at the code, it appears this exception is only used internally (A user should never encounter it in normal operation).  If so, consider making it *internal* which is largely just convention of naming it `_UnpickleableValue` and maybe making a note in the docstring.